data-prepper-plugins/http-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/http/dlq/DlqPushHandler.java

```diff
@@ -60,6 +64,7 @@ public DlqPushHandler(final String dlqFile,
                           final String dlqPathPrefix) {
         if(dlqFile != null) {
             this.dlqFile = dlqFile;
+            this.objectWriter = new ObjectMapper().writer().withDefaultPrettyPrinter();
```

Comment:
I don't think we want to write using the pretty printer. This will result in large files. It is also no longer ndjson.

Reply:
Thanks for review @dlvenable , I shall surely remove prettyPrinter